backend/protocol_rpc/websocket.py

```
@@ -0,0 +1,124 @@
+from fastapi.websockets import WebSocketDisconnect
+
+from backend.protocol_rpc.broadcast import Broadcast
+
+GLOBAL_CHANNEL = "__broadcast__"
+
+
+async def _forward_messages(
+    websocket: WebSocket,
+    subscriber,
+    *,
+    channel: str,
```

Ruff (0.13.1)

Unused function argument: `channel`

(ARG001)